docs/design.md

```diff
@@ -1,317 +1,135 @@
+
+- Service Catalog users can register service brokers with Kubernetes
+- Service Catalog can fetch the services and plans (called the catalog) from 
+each service broker, and make it available to Kubernetes users
+- Kubernetes users can request a new service from a broker by submitting a
+`ServiceInstance` resource to Service Catalog
+- Kubernetes users can link an application (i.e. one or more `Pod`s) to a 
+service by creating a new `ServiceBinding` resource
 
-This infrastructure allows for a loose-coupling between Applications
-running in Kubernetes and the Services they use.
-The Service Broker, in its most basic form, is a blackbox entity. Whether
-it is running within Kubernetes itself is not relevant. This allows for
-the Application that uses those Services to focus on its own business logic
-while leaving the management of these Services to the entity that owns
-them.
+These features provide a loose-coupling between Applications running in 
+Kubernetes and services that they use.
+
+Generally, the services that applications use are external (i.e. "black box")
+to the Kubernetes cluster. For example, applications may decide to use a
+cloud database service. 
+
+Using Service Catalog and the appropriate service broker, application
+developers can focus on their own business logic, leave development and
+management of the service to someone else, and leave provisioning to the 
+Service Catalog and broker.
 
 ## Terminology
 
-- **Application** : Kubernetes uses the term "service" in a different way
+- **Application**: Kubernetes uses the term "service" in a different way
   than Service Catalog does, so to avoid confusion the term *Application*
-  will refer to the Kubernetes deployment artifact that will use a Service
-  Instance.
-- **ServiceBinding**, or *Service Binding* : a link between a Service Instance
-  and an Application. It expresses the intent for an Application to
-  reference and use a particular Service Instance.
-- **ServiceBroker**, or *Service Broker* : a entity, available via a web endpoint,
-  that manages a set of one or more Services.
-- **Credentials** : Information needed by an Application to talk with a
-  Service Instance.
-- **ServiceInstance**, or *Service Instance* : Each independent use of a Service
-  Class is called a Service Instance.
-- **Service Class**, or *Service* : one type of Service that a Service Broker
-  offers.
-- **Plan**, or *Service Plan* : one type of variant of a Service Class. For
-  example, a Service Class may expose a set of Plans that offer
-  varying degrees of quality-of-services (QoS), each with a different
-  cost associated with it.
+  will refer to the Kubernetes deployment artifact that will use a 
+  `ServiceInstance`.
+- **ClusterServiceBroker**: a Kubernetes resource that Service Catalog recognizes.
+  This resource tells Service Catalog to fetch the catalog of a new broker,
+  and merge the new catalog into the existing `ClusterServiceClass`es and 
```

Comment:
I think the "and merge ... into the existing" phrase is a bit confusing - in the normal "simple" situation there are no existing `ClusterServiceClass` and `ClusterServicePlan` objects yet, so I would rather focus on that scenario, rather than the edge case of re-sync.

Reply:
this isn't intended to describe re-sync, it's talking about adding new services to the existing catalog

Reply:
An idea for a rephrase:

>This resource tells service catalog to fetch the catalog of a new broker, and add that catalog to the existing `ClusterServiceClasses`

What do you think?

Reply:
@arschles I am not exactly sure what are you trying to solve with mentioning "existing" there? :) 
I would not mention this at all. Just something like:
> This resource tells service catalog to fetch the catalog of a new broker, and generate corresponding `ClusterServiceClass`es and `ClusterServicePlan`s (see below).

As a software developer, I appreciate your effort to reduce the ambiguity there though :)